index.ts

```diff
@@ -86,10 +147,22 @@ class Message {
 }
 
 function run () {
+  const bouOptions = getBouOptionsFromConfigYaml(CONFIG_FILE)
+  if (!bouOptions) {
+    console.error('Some error occured while reading config file')
+    // TODO 設定ファイルの不備で落ちたら再起動しても落ちるのは明白なので別ロジックにしたい
+    process.exit(1)
+  }
   /**
    * ref1. https://github.com/beebotte/bbt_node/blob/master/lib/stream.js
    * ref2. https://github.com/beebotte/bbt_node/blob/master/lib/mqtt.js
    */
+  if (bouOptions.beebotteChannelToken.length === 0) {
+    console.error("[!] The token doesn't have positive length.")
```

Comment:
(ありえないですが)lengthが負数のチェックをしていないので、単に「長さが0だよ」と言ったほうが良いと思います

```suggestion
    console.error("[!] The length of token is zero")
```